tests/test_smoke.py

```diff
@@ -267,6 +274,7 @@ def test_minimal(generic_cloud: str):
             f'sky logs {name} 2 --status',  # Ensure the job succeeded.
         ],
         f'sky down -y {name}',
+        _get_timeout(generic_cloud),
```

Comment:
Can we revert this `_get_timeout`? It seems this function is only used in the tests that has `no_fluidstack` mark.

Reply:
Oh, makes sense! I misread that. It looks good to me.